File: back/customerinfo/tests/utils/queries.py
def get_all_customers() -> str:
    return """
        query {
            allCustomers {
                id
                firstName
                lastName
                email
                gender
                company
                title
                city {
                    name
                    latitude
                    longitude
                }
            }
        }
    """


def get_customer_by_id(customer_id: int) -> str:
    return (
        """
        query {
            customer (id: %d) {
                id
                firstName
                lastName
                email
                gender
                company
                title
                city {
                    name
                    latitude
                    longitude
                }
            }
        }
        """
        % customer_id
    )

File: front/src/queries.js
import { gql } from "apollo-boost";

export const GET_ALL_CUSTOMERS = gql`
  query {
    allCustomers {
      id
      firstName
      lastName
      gender
      city
    }
  }
`;

export const GET_CUSTOMER_BY_ID = gql`
  query getCustomerById($id: Int){
    customer(id: $id) {
      id
      firstName
      lastName
      email
      gender
      company
      city
      title
      latitude
      longitude
    }
  }
`;
